cartridge-cli.lua

```diff
@@ -2846,13 +3063,74 @@ local function instantiate_template(template_dir, dest_dir, app_name)
         local destdir = fio.dirname(destname)
 
         if not fio.path.exists(destdir) then
-            make_tree(destdir)
+            local ok, err = make_tree(destdir)
+            if not ok then return false, err end
         end
 
-        write_file(destname, content, mode)
+        local ok, err = write_file(destname, content, mode)
+        if not ok then return false, err end
     end
+
+    return true
 end
 
+local function create_app_directory_and_init_git(dest_dir, template, name)
+    assert(fio.path.exists(dest_dir))
```

Comment:
assert?

Reply:
Yes, this function expects that `dest_dir` is already exists.